packages/core/src/baseclient.ts

```diff
@@ -337,13 +338,15 @@ export abstract class BaseClient<O extends ClientOptions> implements Client<O> {
     // Note: we use `event` in replay, where we overwrite this hook.
 
     if (this._options.sendClientReports) {
-      // We want to track each category (error, transaction, session, replay_event) separately
+      const clientReportCategory = dataCategoryToClientReportCategory(category);
```

Comment:
Arguably, it'd be better to change the signature's type to ClientReportCategory but this is breaking. Also, we'd need to call the conversion function more often which increases bundle size